internal/persistence/mongo.go

```diff
@@ -26,7 +26,7 @@ func NewMongoClient(uri string) *MongoClient {
 	return &MongoClient{client: client}
 }
 
-func (mc *MongoClient) NewMongoDB() *mongo.Database {
+func (mc *MongoClient) NewMongoUserDB() *mongo.Database {
```

Comment:
can we simply the name to CreateDB and let the db name be part of the input parameters? In this case we're call the DB users but if this is an exposed function in the package I'd like to inject then name of the db instead of hard code it.

also since this is an explicitly mongo package, I'd just call it mongo instead of persistence so we know what we're calling mongo.CreateDB("user"); reads clearer in my opinion.

Reply:
Fixed in the new commit